JEMS-CSE416/JEMS
# splash screen

Updates:
- [x] Added splash screen with get started button that routes to home screen.

Testing:
1. Click on get started button
2. Confirm that page gets redirected to home screen
